README.md

```diff
@@ -1 +1,89 @@
-# Berry-IMU
\ No newline at end of file
+# Berry-IMU
+This module provides an implementation of the [BerryIMUv3](https://ozzmaker.com/product/berryimu-accelerometer-gyroscope-magnetometer-barometricaltitude-sensor/) using an i2c connection on ARM64 systems.
```

Comment:
Sorry, one last comment! Let's expand this with a direct link to the module's registry page as well:

```suggestion
This module provides an implementation of the [BerryIMUv3](https://ozzmaker.com/product/berryimu-accelerometer-gyroscope-magnetometer-barometricaltitude-sensor/) using an i2c connection on ARM64 systems,
and is [available from the Viam registry](https://app.viam.com/module/viam-labs/berryimu).
```